pkg/clusteragent/autoscaling/workload/telemetry.go

```diff
@@ -25,3 +30,26 @@ var rolloutTriggered = telemetry.NewCounterWithOpts(
 	"Tracks the number of patch requests sent by the patcher to the kubernetes api server",
 	commonOpts,
 )
+
+func startLocalTelemetry(ctx context.Context, sender sender.Sender, tags []string) {
```

Comment:
I'm wondering why not using the datadog_cluster_agent integration to send the telemetry?
is it just temporary?

Reply:
Because we need to ship a new Agent version to do it.